### DISCUSSION TITLE
Cannot start epoptes_1.0.0

### DISCUSSION THREAD

**Discussion by homerlinux** [epoptes/epoptes#78] [Q&A]
I have epoptes_1.0.0-1+t201809210732~ubuntu18.04.1_all.deb installed on Linux Mint 18.3 (from the ppa:epoptes/ppa repository)

Whenever I execute /usr/bin/epoptes, I get the following error:

Traceback (most recent call last):
  File "/usr/bin/epoptes", line 99, in <module>
    main()
  File "/usr/bin/epoptes", line 89, in main
    egui = gui.EpoptesGui()
  File "/usr/lib/python3/dist-packages/epoptes/ui/gui.py", line 83, in __init__
    self.get('mi_remote_assistance').set_property('visible', False)
AttributeError: 'NoneType' object has no attribute 'set_property'

Note: downgrading "epoptes" to version 0.5 workarounds the issue, but thumbnails are not shown when clients are running version 1.0.0

Note 2: commenting out line 83 from "/usr/lib/python3/dist-packages/epoptes/ui/gui.py" seems to do the trick and everything looks good

**Comment by alkisg** [comment]
This is a duplicate of https://github.com/Epoptes/epoptes/issues/75
